app/workers/aggregation_completed_mailer_worker.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AggregationCompletedMailerWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :data_high
+
+  def perform(agg_id)
+    aggregation = Aggregation.find(agg_id)
+    AggregationCompletedMailer.aggregation_complete(aggregation).deliver
```

Comment:
I think `.deliver` might be a deprecated method replaced by `deliver_now` or `deliver_later`

Reply:
apparently so! Every other mailer uses it, but I'll just change it here and leave that refactor for later.